src/test/java/com/redhat/darcy/webdriver/FirefoxBrowserFactoryTest.java

```diff
@@ -37,6 +38,7 @@
     @Before
     public void checkForDriver() {
         assumeTrue(System.getProperty("java.class.path").contains("firefox-driver"));
+        assumeNotNull(System.getProperty("webdriver.gecko.driver"));
```

Comment:
The FirefoxBrowserFactoryTest does run and pass if you run it in an environment with the geckodriver binary installed.  I got the impression that wasn't possible in travis environment, so I treated it the same as ChromeDriver, IEDriver, etc.  Happy to fix it if I misunderstood.